complex/ft_config.c

```diff
@@ -40,8 +41,38 @@
 #define FT_MODE_ALL	/*FI_CONTEXT |*/ FI_LOCAL_MR /*| FI_MSG_PREFIX*/
 #define FT_MODE_NONE	~0ULL
 
+#define STRINGIFY(x) #x
+#define STREQ(STR1, STR2) !strncmp(STR1, STR2, strlen(STR2))
```

Comment:
Same reasoning  as ftset_sizeof macro below but if the meaning isn't obvious, I can drop the macros.